<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fuzzy Intelectual Interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #44596e;
	font: 11pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: #44596e;
	font: 11pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;MOST COMFORTABLE ROUTE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: #44596e;
	font: 14pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
           </property>
           <property name="text">
            <string>Distance</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dist_line">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	    margin-top: 5px;          
        border-radius: 6px;
        padding-left: 5px;
		text-align: center;
	font: 14pt &quot;Open Sans&quot;;
}</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: #44596e;
	font: 14pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
           </property>
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="price_line">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	    margin-top: 5px;          
        border-radius: 6px;
        padding-left: 5px;
		text-align: center;
	font: 14pt &quot;Open Sans&quot;;
}</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: #44596e;
	font: 14pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
           </property>
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="time_line">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	    margin-top: 5px;          
        border-radius: 6px;
        padding-left: 5px;
		text-align: center;
	font: 14pt &quot;Open Sans&quot;;
}</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: #44596e;
	font: 14pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
           </property>
           <property name="text">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="space_line">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	    margin-top: 5px;          
        border-radius: 6px;
        padding-left: 5px;
		text-align: center;
	font: 14pt &quot;Open Sans&quot;;
}</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox">
           <property name="styleSheet">
            <string notr="true">QComboBox{
	    margin-top: 5px;          
        border-radius: 6px;
        padding-left: 5px;
		text-align: center;
	font: 12pt &quot;Open Sans&quot;;
}</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_2">
           <property name="styleSheet">
            <string notr="true">QComboBox{
	    margin-top: 5px;          
        border-radius: 6px;
        padding-left: 5px;
		text-align: center;
	font: 12pt &quot;Open Sans&quot;;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: #44596e;
	font: 14pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
           </property>
           <property name="text">
            <string>Point 1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: #44596e;
	font: 14pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
           </property>
           <property name="text">
            <string>Point 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #25d366;
	font: 11pt &quot;Open Sans&quot;;
	color: #ffffff;       
}</string>
           </property>
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #eb4924;
	font: 11pt &quot;Open Sans&quot;;
	color: #ffffff;
}</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: #1e88e5;
	font: 11pt &quot;Open Sans&quot;;
	color: #ffffff;
	border-top-left-radius: 20px;
	border-top-right-radius: 20px;
	border-bottom-left-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Comfort Level&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: #1e88e5;
	font: 18pt &quot;Open Sans&quot;;
	text-align: center;
	color: #ffffff;
	border-top-left-radius: 20px;
	border-top-right-radius: 20px;
	border-bottom-left-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
